pkg/interface/CONTRIBUTING.md

```diff
@@ -26,130 +27,17 @@ module.exports = {
 };
 ```
 
-This file is the configuration file for your front-end development environment.
-Let's walk through it.
-
-The first line, listing piers, specifies which piers to copy the JS files into.
-By default, the development environment won't copy files into any pier, even if
-you've set the pier in `urbitrc`.
-
-If you want to copy the JS files into your ship, as it would run in a regular
-user environment, uncomment these lines in
-`pkg/interface/config/webpack.dev.js`:
-
-```javascript
-// uncomment to copy into all piers
-//
-// return Promise.all(this.piers.map(pier => {
-//   const dst = path.resolve(pier, 'app/landscape/js/index.js');
-//   copyFile(src, dst).then(() => {
-//     if(!this.herb) {
-//       return;
-//     }
-//     pier = pier.split('/');
-//     const desk = pier.pop();
-//     return exec(`herb -p hood -d '+hood/commit %${desk}' ${pier.join('/')}`);
-//   });
-// }));
-```
-
-And then set your pier in `urbitrc` (ensure it ends in `/home`). The `herb`
-option in your `urbitrc` will automatically commit the changes to your ship if
-you have herb installed (see `pkg/herb`).
-
-For most developers, if you are making changes to Landscape without any back-end
-changes on the Urbit ship itself, and you have an Urbit ship running already,
-you don't have to boot a development ship. You can simply set up the dev server
-for the development environment and point it at your running ship.
-
-To do this, set the `URL` property in your urbitrc and replace it with the URL
-of the urbit that you are testing on. For example, a development ship by default
-lives at `localhost:80` so our `urbitrc` would have:
-
-```javascript
-module.exports = {
-  URL: 'http://localhost:80'
-}
-```
-
-Then get everything installed:
+Change the URL to your livenet ship (if making front-end changes) or keep it the
+same (if [developing on a local development ship][local]). Then, from
```

Comment:
My local ships have never bound to `80` because I have something else running locally. Might want a sidenote here about using whatever port the ship prints on boot.

Reply:
*1 minute ago* god damn it mark

Reply:
I'm willing to accept the tradeoff that anyone with something else running locally knows that ports exist